app/packages/spotlight/examples/index.html

```
@@ -0,0 +1,83 @@
+        flex: 1;
+        min-height: 0;
+        --fo-palette-background-level1: #2a2a2a;
+        --fo-palette-text-tertiary: #555;
+      }
+    </style>
+  </head>
+  <body>
+    <div id="toolbar">
+      <div id="toolbar-top">
+        <h1>@fiftyone/spotlight ✨</h1>
+        <button id="btn-vertical" class="active">Vertical</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)